webapp/src/App.vue

```diff
@@ -19,6 +19,11 @@
                 >Events</router-link
               >
             </li>
+            <li class="nav-item" v-if="profile">
+              <router-link :to="{ name: 'Service' }" class="nav-link"
+                >Dienste</router-link
+              >
+            </li>
           </ul>
           <profile-nav></profile-nav>
```

Comment:
`ProfileNav` also fetches the profile endpoint. We could forward the fetched value as a prop to avoid fetching it twice.

Reply:
It is cached in the profile.ts, therefore it is only fetches once